exonum/src/encoding/error.rs

```diff
@@ -101,7 +101,7 @@ pub enum Error {
         /// start of new segment
         start: Offset,
     },
-    /// Between segments foud spaces
+    /// Between segments found spaces
```

Comment:
If I understand the meaning correctly, I would suggest to rephrase as "Spaces found between segments"